src/cmake/modules/FindR3DSDK.cmake

```diff
@@ -0,0 +1,68 @@
+# - Try to find R3D SDK
+#
+#  Set RED_ROOT Environment variable to root of the SDK
+#  Otherwise it will try to guess some plausible paths.
+#
+# Once done, this will define
+#
+#  RED_FOUND - system has R3D SDK
```

Comment:
I think that to avoid confusion, we should match the find_package(R3DSDK) with the names of these variables: R3DSDK_FOUND, etc. Across the board. I can tell you from experience that it's super confusing to mix two different package nomenclatures.

Reply:
Done!